src/tasks/task-bholdus-bsc/services/bsc-header-listener.service.ts

```diff
@@ -1,11 +1,82 @@
 import { inject, injectable } from 'inversify';
 import { IService } from '@lib/task';
 import { BholdusBscTaskBus } from '../bus';
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import getApiOptions from '@bholdus/api-options';
+import { ethers } from 'ethers';
+import { BholdusWriterMessage } from '../message'
+import { SubjectLike, Subject } from 'rxjs';
+import '@bholdus/types'
+import { BscPrimitivesBscHeader } from '@bholdus/types/interfaces'
+import { Bytes } from '@polkadot/types';
+
 
 @injectable()
 export class BscHeaderListenerService implements IService<BholdusBscTaskBus> {
 	static symbol = Symbol(BscHeaderListenerService.name);
 	@inject(BholdusBscTaskBus.symbol) private bus: BholdusBscTaskBus;
 
-	async start(): Promise<void> {}
+	async start(): Promise<void> {
+		
+		const provider = new ethers.providers.JsonRpcProvider("https://bsc-dataseed1.binance.org:443");
+
+		const wsProvider = new WsProvider("ws://127.0.0.1:9944");
+		const api = await ApiPromise.create(getApiOptions({ provider: wsProvider }));
+
+		const latestBlock = await api.query.bsc.finalizedCheckpoint();
+
+		const blockNumber = Number(latestBlock.number.toBigInt() + BigInt("200"));
+
+		let epockBlock = await this.getBlock(blockNumber, api, provider);
+
+		const subject = this.bus.channel(BholdusWriterMessage);
+
+		// console.log(epockBlock);
+
+		let listBlock = [epockBlock];
+
+		for (let index = 1;index < (epockBlock.get("extraData").toU8a().length + 1)/2; index++) {
```

Comment:
Does `toU8a` really returns number of validators?. `toU8a` should return an array of bytes and its length is the number of bytes in the array, not the number of validators?. The number of validators on BSC is 21, maybe we should set this number as a constant